### PR TITLE
Fix typos in Javadoc

### PR DESCRIPTION
**Pull request by nodmp**
### Contributor Checklist
- [x] [Review Contribution Guidelines](https://github.com/gradle/gradle/blob/master/CONTRIBUTING.md)
- [x] Make sure that all commits are [signed off](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff) to indicate that you agree to the terms of [Developer Certificate of Origin](https://developercertificate.org/).
- [x] Make sure all contributed code can be distributed under the terms of the [Apache License 2.0](https://github.com/gradle/gradle/blob/master/LICENSE), e.g. the code was written by yourself or the original code is licensed under [a license compatible to Apache License 2.0](https://apache.org/legal/resolved.html).
- [x] Check ["Allow edit from maintainers" option](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) in pull request so that additional changes can be pushed by Gradle team
~- [x] Provide integration tests (under `<subproject>/src/integTest`) to verify changes from a user perspective~
~- [x] Provide unit tests (under `<subproject>/src/test`) to verify logic~
- [x] Update User Guide, DSL Reference, and Javadoc for public-facing changes
- [x] Ensure that tests pass sanity check: `./gradlew sanityCheck`
- [x] Ensure that tests pass locally: `./gradlew <changed-subproject>:quickTest`

### Reviewing cheatsheet

Before merging the PR, comments starting with 
- ❌ ❓**must** be fixed
- 🤔 💅 **should** be fixed
- 💭 **may** be fixed
- 🎉 celebrate happy things

### CODE REVIEW

File: subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/plugins/ide/tooling/r33/BuildActionCompositeBuildCrossVersionSpec.groovy [hunk 1]
```diff
@@ -22,16 +22,16 @@ import org.gradle.integtests.tooling.fixture.ToolingApiVersion
...
  * combinations of versions of the TAPI and Gradle.
  *
  * Support for clients using a tooling API version older than 3.0 was removed in Gradle 5.0, so
  * there is a class-level lower bound for both versions.
  *
  * In addition, in version 6.6 the deprecated {@code DependencySubstitutions#with(ComponentSelector)} method
  * was removed, to be replaced by {@code #using(ComponentSelector)} so there are 2 versions of tests
- * present using either of those methods - pre and post Gradle 6.6.  And when the TAPI or Gradel versions
- * are overriden on the method level, you have to be sure to copy the lower bound down to the method level
+ * present using either of those methods - pre and post Gradle 6.6.  And when the TAPI or Grade versions
```
Comment (eskatos) [inline subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/plugins/ide/tooling/r33/BuildActionCompositeBuildCrossVersionSpec.groovy:33]: :x: fix a typo in the typo fix :)
```suggestion
 * present using either of those methods - pre and post Gradle 6.6.  And when the TAPI or Gradle versions
```

**Review by eskatos — CHANGES_REQUESTED** [review]

File: subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/plugins/ide/tooling/r33/BuildActionCompositeBuildCrossVersionSpec.groovy (hunk 1 above)
Comment (nodmp) [reply subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/plugins/ide/tooling/r33/BuildActionCompositeBuildCrossVersionSpec.groovy]: Done.

**Review by eskatos — APPROVED** [review]

**Comment by eskatos** [comment]
@bot-gradle test and merge

**Comment by eskatos** [comment]
@bot-gradle test and merge

**Pull request merged**